documentation/docs/scenarios/upload-to-transportrequest/Readme.md

```diff
@@ -0,0 +1,197 @@
+# Build an SAP Fiori Application and attach it to a transport request on an ABAP system with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and attach the build result to a transport request in an SAP ABAP system.
+
+There are in principle two technical ways how to attach a binary to an ABAP transport request. We support an upload via RFC and an upload via OData. Which option to use depends on the version of your ABAP system. For AS ABAP 7.50 SP08 or 7.51 SP07 or 7.52 SP03 or newer the OData based upload should be used, for older versions the RFC based upload needs to be used.
+
+## Prerequisites
+
+* [Docker environment](https://docs.docker.com/get-started/)
+* You have set up project “Piper”. See [guided tour](../../../guidedtour/)
+* A transport request
```

Comment:
A transport request can be opened also on the fly. It is not a "hard" prerequisite. Or at least we should document the two possibilities here

Reply:
@IKirilova a transport request can also be created within the pipeline `transportRequestCreate`. But the scenario here describes the case without pre-existing transport request. I guess it is for the sake of didactical reduction. The question is: should we make the scenario here more complex and also explain how to act without pre-existing transport request. I added a sentence inside the pre-requisites section explaining that a transport request can also be created in the pipeline.

@SarahNoack : what do you think: is it from the didactical point of view better to omit some details (here: we can also create a transport request during the pipeline run) or should we explain more details?